### PR TITLE
docs: Include lido-status command

### PR DESCRIPTION
**Pull request by khalifaa55**
## Changes:
- Include `sedge lido-status` command in Sedge documentation.
## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe): 

## Testing
**Requires testing**

- [ ] Yes
- [ ] No

**In case you checked yes, did you write tests?**

- [ ] Yes
- [ ] No

### CODE REVIEW

File: docs/docs/quickstart/lido.mdx [hunk 1]
```diff
@@ -104,7 +104,30 @@ meantime.
...
+This command helps you monitor your Node Operator's performance by displaying:
+
+- Node Operator ID
+- Queue information (e.g., available for deposit, stuck, refunded, exited, deposited)
+- Bond and rewards details
+- Alerts for penalties and exit requests
+
+This information assists you in managing your Node Operator effectively and addressing any issues that may arise.
+
+### Additional Management Tasks
```
Comment (AntiD2ta) [inline docs/docs/quickstart/lido.mdx:128]:
```suggestion
- Bond and rewards details

### Additional Management Tasks
```

File: docs/docs/quickstart/lido.mdx [hunk 2]
```diff
@@ -104,7 +104,30 @@ meantime.
...
 Once your validator keys are deposited and your node is running, you can manage your Node Operator using the Lido
-CSM Widget. This interface allows you to:
+CSM Widget and Sedge commands.
+
+### Checking Node Operator Status
+
+To view detailed data about your Node Operator, use the `sedge lido-status` [command](../commands/lidoStatus.mdx). This command displays information specific to the Node Operator based on the provided rewards address.
+
+```bash
+sedge lido-status <your_address>
```
Comment (AntiD2ta) [inline docs/docs/quickstart/lido.mdx:114]:
```suggestion
sedge lido-status <your_reward_address>
```

File: docs/docs/quickstart/lido.mdx [hunk 3]
```diff
@@ -104,7 +104,30 @@ meantime.
 ## Managing Your Node Operator
 
 Once your validator keys are deposited and your node is running, you can manage your Node Operator using the Lido
-CSM Widget. This interface allows you to:
+CSM Widget and Sedge commands.
+
+### Checking Node Operator Status
```
Comment (AntiD2ta) [inline docs/docs/quickstart/lido.mdx:109]: Please add more information regarding the reward address. It should be clear to the user what address to use as the command argument.

**Review by AntiD2ta — CHANGES_REQUESTED** [review]
Made a first review. This PR should change with #402 but looks mostly correct.

**Comment by khalifaa55** [comment]
@AntiD2ta changes addressed [here](https://github.com/NethermindEth/sedge/pull/403/commits/8d66c2cfef87fa833a45b824dfc3aeaa0726841d).

**Review by AntiD2ta — APPROVED** [review]

**Pull request merged**